### ISSUE TITLE
New license request: VL [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by wurzer** [spdx/license-list-XML#2557]
**1.** License Name: Volla License 2.0
**2.** Short identifier: VL
**3.** License Author or steward: Dr. Jörg Wurzer
**4.** Comments: The License allows developer to contritbute, fork and use the source code for won use. Criteria A, C, D and E is fulfilled.
**5.** License Request Url: http://tools.spdx.org/app/license_requests/388
**6.** URL(s): https://github.com/HelloVolla/android-launcher-qt/blob/master/LICENSE.txt
**7.** OSI Status: Not Submitted
**8.** Example Projects: https://github.com/HelloVolla

**Comment by swinslow** [comment]
@wurzer Can you please share a link to projects other than those at https://github.com/HelloVolla that are using this license?

**Comment by karsten-klein** [comment]
**{metæffekt} Universe**
no match

markers: No Warranty Marker, **Non-commercial Marker**

**ScanCode**
matched id: [other-permissive](https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/other-permissive.LICENSE)
matched id: [proprietary-license](https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/proprietary-license.LICENSE)

**Comment**
No open source license due to the limits in use. No evidence for significant use. We will add the license to the universe for the sake of risk management. No vote for adding to SPDX license list regarding current guidelines.

**Comment by jlovejoy** [comment]
I generally find it a bit troublesome that this license looks a lot like MIT in terms of layout and matching text, but obviously has some significant differences. Use of the license could easily confuse people who didn't read closely, which is not good for the ecosystem.

If it had a bunch of use, this could almost be a reason to add it to make it more easily identifiable/distinguishable, but given it does not seem to have substantial use, I would not be in favor of adding it to SPDX License List at this time.

@karsten-klein @richardfontana - is that where you both ended up? Hard to tell from your comments above.

**Comment by Pizza-Ria** [comment]
That is definitely NOT an open source license.  Agree with @karsten-klein although I'm sympathetic to @jlovejoy's point as well - the lack of substantial usage is the tie-breaker for me.  So, I'll vote to not add to SPDX due to the lack of usage.

**Comment by swinslow** [comment]
Agreed, no substantial use documented here beyond the project linked above. From a quick Google search, it looks like this "HelloVolla" org is the only one using this license.

Given the non-open source nature of the license, together with limited and non-substantial use, this would not satisfy the balance of the "Other factors" in the [license inclusion principles](https://github.com/spdx/license-list-XML/blob/main/DOCS/license-inclusion-principles.md). Based on this and the other comments above, I'm going to close this issue.

**Comment by wurzer** [comment]
The Volla license was developed by Volla Systeme GmbH, my company, because we have published and will publish the source code of our Android operating system and some apps on https://github.com/HelloVolla. In the meantime, our partner is also working with the license: https://github.com/holochain-apps/relay

I have made the license as simple as possible because I prefer pragmatism and comprehensibility to long legal texts. Our patent and trademark law firm had checked and approved the license.

Registering the license here is important for us at Volla because we provide apps via F-Droid. A license can only be referenced there if it is listed here. I want to offer our users transparency, which I consider to be very important. 

The Volla license is used in software that is freely available and that developers can use for non-commercial, personal purposes. You will see that there are some forks of our open source projects, in the spirit of open source. 

There is a lot to argue about open source. I want to avoid this discussion at this point because it is not expedient. Different opinions are possible here.  In the working group for open source of the Gesellschaft für In formatik, I had promoted my point of view on this: https://ak-oss.gi.de/mitteilung/quelloffene-software-als-motor-fuer-innovation-und-vertrauen-einer-neuen-smartphone-marke

I ask you to accept the license in order to create transparency and clarity for our users and developers. The requirement for F-Droid is particularly critical. It is then still up to each user and developer to evaluate the license in detail.
